cabot/urls.py

```diff
@@ -140,6 +140,9 @@
 
      url(r'^admin/', include(admin.site.urls)),
 
+     # for serving static assets using django
+     url(r'^static/(?P<path>.*)$', 'django.views.static.serve', {'document_root': settings.STATIC_ROOT,}),
```

Comment:
In theory if you are using a webserver like nginx, you can skip this completely for non-heroku deploys. Should that be documented if it already isn't? I think no, but up to you.